doc/cookbook/source/conf.py

```diff
@@ -208,6 +208,7 @@
         ('lua', 'lua'),
         ('csharp', 'cs'),
         ('cpp', 'cpp'),
+        ('scala','scala'),
```

Comment:
Maybe we can leave the cookbook out for now, and add scala there in a second attempt (we can just rename the tab to "java/scala" in fact)